microcosm_pubsub/codecs.py

```diff
@@ -15,6 +15,25 @@
 DEFAULT_MEDIA_TYPE = "application/json"
 
 
+def enrich(error: ValidationError, schema: Schema):
+    """
+    Enrich a schema validation error with a human-interpretable
+    description of a marshmallow schema.
+
+    """
+    messages = error.messages
+    if isinstance(messages, dict):
+        messages.update(dict(
+            schema=str(schema),
+        ))
+    else:
+        messages.append(
+            str(schema),
+        )
+
+    return error
```

Comment:
note that:
1. marshmallow ValidationError uses a `messages` list/dict rather than a `message` str
2. we return the same error object we got passed in, and update its error messages in-place. This is so we dont lose additional meta-data that may have been passed to the original error's [constructor](https://marshmallow.readthedocs.io/en/stable/api_reference.html#marshmallow.exceptions.ValidationError).